src/main/java/com/factly/dega/service/OrganizationService.java

```diff
@@ -62,4 +63,14 @@
      */
     Optional<OrganizationDTO> findBySlug(String slug);
 
+    /**
+     * Get the organizations by keycloakId and pageable.
+     *
+     * @param keycloakId the keycloakId of the User
+     * @param pageable the pagination information
+     *
+     * @return List<OrganizationDTO> List of Organizations by keycloakId and pageable
+     */
+    List<OrganizationDTO> getOrganizations(String keycloakId, Pageable pageable);
```

Comment:
rename keycloakId to keycloakUserId as it can be confused for organization's keycloakId

Reply:
done